### PR TITLE
Ot 63 add slides to public endpoint

### PR DESCRIPTION
**Pull request by maxisosa89**
Description

AS user
I WANT to see Slides when I enter the page
TO be more informed about them

Criteria of acceptance:

In the public data endpoint of the Organization, the list of Slides ordered by the "order" field must be added.

The Slides will be those belonging to the Organization, based on the organization_id field that should correspond to the Organization that is being returned

image

### CODE REVIEW

**Comment by maxisosa89** [comment]
Link a JIRA: https://alkemy-labs.atlassian.net/browse/OT291-63

File: dao/slide.js [hunk 1]
```diff
@@ -19,6 +19,19 @@ class SlidesDap {
       throw new Error(error);
     }
   }
+  static async findSlidebyOrganization({ where, exclude = [], order = [] }) {
+    try {
+      console.log(where)
```
Comment (devmontini) [inline dao/slide.js]: Quitar "console.log"

**Comment by devmontini** [comment]
Y ver conflicto del dao

**Comment by maxisosa89** [comment]
Removido el console.log y resuelto el conflicto.

**Review by devmontini — APPROVED** [review]
lgtm

File: controllers/organization.js [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
+      });
+      const slides = await SlidesDao.findSlidebyOrganization({
+        where: { organizationId: response.id },
+        exclude: ['createdAt', 'updatedAt', 'deletedAt'],
+        order: ['order', 'ASC'],
       });
       response === null
         ? res.status(404).json({ msg: 'Could not find information' })
-        : res.status(200).json(response);
+        : res.status(200).json({ response, slides });
```
Comment (soyarielruiz) [inline controllers/organization.js:18]: nit: en vez de response usaría el nombre organization.

File: dao/slide.js [hunk 3]
```diff
@@ -19,6 +19,18 @@ class SlidesDap {
...
+  static async findSlidebyOrganization({ where, exclude = [], order = [] }) {
+    try {
+      const SlideData = await Slide.findAll({
+        where: where,
+        attributes: { exclude: exclude },
+        order: [order],
+      });
+      return SlideData;
+    } catch (error) {
+      throw new Error(error);
```
Comment (soyarielruiz) [inline dao/slide.js:31]: nit: Se pueden agregar excepciones custom para estos casos.

**Review by soyarielruiz — APPROVED** [review]

**Pull request merged**